build-farm/platform-specific-configurations/linux.sh

```diff
@@ -171,5 +170,15 @@ then
 	export CXX=$RISCV64/bin/riscv64-unknown-linux-gnu-g++
 	CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --disable-ddr --openjdk-target=riscv64-unknown-linux-gnu --with-sysroot=/opt/fedora28_riscv_root --with-boot-jdk=$JDK_BOOT_DIR --with-build-jdk=$BUILDJDK"
 	BUILD_ARGS="${BUILD_ARGS} -F"
+elif [ "${ARCHITECTURE}" == "riscv64" && "${VARIANT}" == "${BUILD_VARIANT_BISHENG}" ]; then
+  echo Bisheng RISCV cross-compilation ... 
+  export RISCV64=/opt/riscv_toolchain_linux
+  export LD_LIBRARY_PATH=$RISCV64/lib64
+  export PATH="$RISCV64/bin:$PATH"
+  # riscv has to use a cross compiler
+  export CC=$RISCV64/bin/riscv64-unknown-linux-gnu-gcc
+  export CXX=$RISCV64/bin/riscv64-unknown-linux-gnu-g++
+  CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --openjdk-target=riscv64-unknown-linux-gnu --with-sysroot=/opt/fedora28_riscv_root --with-boot-jdk=$JDK_BOOT_DIR"
+  BUILD_ARGS="${BUILD_ARGS} -F"
```

Comment:
The whitespace in this block doesn't match the above, can you check tabs vs spaces and see which block needs changing?

Reply:
The above code block uses tabs, I replaced it with spaces.